aboutlibraries-core/src/main/java/com/mikepenz/aboutlibraries/Libs.kt

```diff
@@ -548,6 +548,7 @@ class Libs(
     companion object {
         const val BUNDLE_TITLE = "ABOUT_LIBRARIES_TITLE"
         const val BUNDLE_EDGE_TO_EDGE = "ABOUT_LIBRARIES_EDGE_TO_EDGE"
+        const val BUNDLE_SEARCH_ENABLED = "ABOUT_LIBRARIES_SEARCH_ENABLED"
```

Comment:
This is a reminder for myself. We should move the non core related flags out from the core module